hr_recruitment_security/security/security.xml

```diff
@@ -0,0 +1,23 @@
+<odoo>
+    <data>
+        <record id="group_hr_recruitment_officer_own_documents" model="res.groups">
+            <field name="name">Officer (own documents)</field>
+            <field name="category_id" ref="base.module_category_hr_recruitment"/>
+            <field name="implied_ids" eval="[(4, ref('hr_recruitment.group_hr_recruitment_user'))]"/>
+        </record>
+        <record model="ir.rule" id="hr_job_group_hr_recruitment_officer_own_documents_rule">
+            <field name="name">View job positions (own responsible)</field>
+            <field name="model_id" ref="hr_recruitment.model_hr_job" />
+            <field name="global" eval="True" />
+            <field name="domain_force">[('hr_responsible_id', '=', user.id)]</field>
+            <field name="groups" eval="[(4,ref('hr_recruitment_security.group_hr_recruitment_officer_own_documents'))]" />
```

Comment:
Use relative ref so it's not affected by module name changes:
```suggestion
            <field name="groups" eval="[(4,ref('group_hr_recruitment_officer_own_documents'))]" />
```

Reply:
Thanks for the aclaration about references